--- a/stdlib/builtins.pyi
+++ b/stdlib/builtins.pyi
@@ -3118,8 +3118,8 @@ class dict(MutableMapping[_KT, _VT]):
     @overload  # type: ignore[override]
-    def get(self, key: _KT, default: None = None, /) -> _VT | None:
+    def get(self, key: object, default: None = None, /) -> _VT | None:
         """Return the value for key if key is in the dictionary, else default."""
     @overload
-    def get(self, key: _KT, default: _VT, /) -> _VT: ...
+    def get(self, key: object, default: _VT, /) -> _VT: ...
     @overload
-    def get(self, key: _KT, default: _T, /) -> _VT | _T: ...
+    def get(self, key: object, default: _T, /) -> _VT | _T: ...
 
