--- a/stdlib/_collections_abc.pyi
+++ b/stdlib/_collections_abc.pyi
@@ -84,3 +84,3 @@ class dict_keys(KeysView[_KT_co], Generic[_KT_co, _VT_co]):  # undocumented
     if sys.version_info >= (3, 13):
-        def isdisjoint(self, other: Iterable[_KT_co], /) -> bool:
+        def isdisjoint(self, other: object, /) -> bool:
             """Return True if the view and the given iterable have a null intersection."""
@@ -108,3 +108,3 @@ class dict_items(ItemsView[_KT_co, _VT_co]):  # undocumented
     if sys.version_info >= (3, 13):
-        def isdisjoint(self, other: Iterable[tuple[_KT_co, _VT_co]], /) -> bool:
+        def isdisjoint(self, other: object, /) -> bool:
             """Return True if the view and the given iterable have a null intersection."""
